packages/cadl-ranch-specs/http/models/input-basic/main.cadl

```diff
@@ -24,6 +24,6 @@ model InputModel {
 
 @scenario
 @scenarioDoc("Generate and send an input-only model with required reference and value type properties.")
-@route("/models")
+@route("/models/basic")
 @get
 op getModel(@body input: InputModel): Response<200> | ErrorResponse ;
```

Comment:
is `Response<200>` same as `OkResponse`?

Reply:
Yes, https://github.com/microsoft/cadl/blob/main/packages/rest/lib/http.cadl#L35